docs/learn/deploy/3b-expfactory.rst

```diff
@@ -0,0 +1,141 @@
+.. _tutorial/deploy/third-party/expfactory:
+
+The Experiment Factory
+======================
+
+`The Experiment Factory`_, referred to as "expfactory," is is an open source infrastructure
+for building and deploying reproducible experiment containers. You can export a LabJS
+experiment (or more than one!) and build it into a container (with other experiments,
+not all of which need to be LabJS) that is ready for deployment on a webserver with technologies
+like SSL, and databases ranging from filesystem (.json), to sqlite, to postgres and sql. 
+
+.. _Expfactory: https://expfactory.github.io
+.. _Documentation: https://expfactory.github.io/expfactory/integration-labjs
+.. _Expfactory-LabJS Builder on Github: https://www.github.com/expfactory/builder-labjs
+.. contents:: Steps
+  :local:
+
+----
+
+Design Your Experiment
+-----------------------
+
+Take a look at how to `get started <https://labjs.readthedocs.io/en/latest/learn/builder/index.html>`_ 
+with the LabJS builder interface, located at `https://labjs.felixhenninger.com/`_. This
+is where you will design your experiment. When you finish, in order to use it with the Experiment
+Factory, you can select this export option:
+
+.. image:: 3b-expfactory/export.png
+
+This will export a zip file of all the files needed to plug into the Experiment Factory! To help you learn and get started, we provide an example export of a Stroop task in the `builder-labjs repository`<https://www.github.com/expfactory/builder-labjs>_.  We will walk through the basic steps to get this experiment running in your experiment container, and you can do the same with your own exported experiment.
+
+
+Create a Repository
```

Comment:
I'm currently working through this, and this point got me thinking -- would it make sense for the `lab.js` export to already add the `.circleci` folder?